lib/cmd/common/apps.js

```diff
@@ -47,7 +47,7 @@ function nguiApps(args, cb) {
     return doRead(projectId, appId, cb);
   }else if (action === 'create'){
     args.shift();
-    return create(args, cb);
```

Comment:
@david-martin I thought about that but it looks like all the unit tests correctly pass the ```{ _: args }``` format

I think that maybe this needs to be more fleshed out?
https://github.com/feedhenry/fh-fhc/blob/master/test/accept/test_as_cli.js